docs-source/content/samples/simple/domains/model-in-image/update3.md

```diff
@@ -93,7 +93,11 @@ Here are the steps for this use case:
      Now, you use the Image Tool to create an image named `model-in-image:WLS-v2` that's layered on a base WebLogic image. You've already set up this tool during the prerequisite steps.
 
      Run the following commands to create the model image and verify that it worked:
-
+     
+     {{% notice note %}}
```

Comment:
This note does not appear correctly in the UI, https://oracle.github.io/weblogic-kubernetes-operator/samples/simple/domains/model-in-image/update3/. My guess is that it is indented too much.